available_images.md

```diff
@@ -164,6 +175,34 @@ Tensorflow(v1) native backend from version 23.04 onwards.
 |NVIDIA Triton Inference Server 21.08    |inference	|No			|GPU 		| 3.8 (py38)			|007439368137.dkr.ecr.us-east-2.amazonaws.com/sagemaker-tritonserver:21.08-py3		|
 |NVIDIA Triton Inference Server 21.08    |inference	|No			|CPU 		| 3.8 (py38)			|007439368137.dkr.ecr.us-east-2.amazonaws.com/sagemaker-tritonserver:21.08-py3-cpu		|
 
+For Tritonserver versions 21.08 through 23.12, please use the following account_id map to obtain the correct account id to pull the container from, based on the region:
+```
+account_id_map = {
```

Comment:
is there a doc or link you can provide to these old mappings?

Reply:
If not, any reason to add them in now? It may be more confusing as there are a lot of references to accounts, and you already have the pysdk snippet above that may serve as a better source of info about account IDs for a specific region and version

Reply:
Added them to make it clear between old and new accounts, but I agree, this can be confusing, instead I've just linked to an old 'example'